Git-Collabration-Example/ViewController.swift

```diff
@@ -6,14 +6,53 @@
 //
 
 import UIKit
+import PhotosUI
 
 class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
-    }
-
+        
+        navigationItem.rightBarButtonItems = [
+            UIBarButtonItem(barButtonSystemItem: .camera, target: self, action: #selector(showAlbum)),
+            UIBarButtonItem(barButtonSystemItem: .refresh, target: self, action: nil)
+        ]
 
+    }
+    
+    @objc func showAlbum() {
+        var config = PHPickerConfiguration()
+        config.selectionLimit = 1
```

Comment:
Do you plan to increase the limit later in the code?

Reply:
I might change it in the future but I still need some testing with the code.